app/src/main/res/values/strings.xml

```diff
@@ -10,6 +10,7 @@
     <string name="vessel_name">Vessel Name</string>
     <string name="vessel">Vessel</string>
     <string name="vessel_information">Vessel Information</string>
+    <string name="add_an_attachment">Add Attachments</string>
```

Comment:
Change to "Add Photo or Note" instead of "Add Attachments". It's more clear about what you are doing when you click the button.

Reply:
Makes sense. Updated the string values.